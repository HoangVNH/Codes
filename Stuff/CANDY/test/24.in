10
25
VVVVVVVDDDDVVVVVVVVDVDDDD
49
DXXVVVVVDDDDDDVVVXXXXXDDDDXXXXXXXXVDDDDDDDDDXXXDD
22
DVVVVVXXXXDDDXXVVVVVVV
42
DVVVXDDDVVVVVVVVVVVVVVVVVVVVVVVVVVXXXDDDDV
35
VVVVVXXDDDXXXXXXXXXDDXXXXXDDDDDDDDD
33
XXXXXXXXXXXXVVVVVDVVVVVDDDDDXXXXD
20
VVVXXXXXXXXVVVVVVVVV
21
VVVVXXXXXXDDDXXXXVVVV
33
VVVVVVVVXXXXXDDDDDDDDDVVVDDDDDXXX
42
VVVVVXXXDDDDDDDDDDVVVVVVVDDDDDDXXXDDDDXXXX
