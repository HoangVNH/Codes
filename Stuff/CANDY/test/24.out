24
48
20
40
34
32
18
20
32
40
